yarn.lock

```diff
@@ -8956,6 +8968,23 @@ eslint-cli@1.1.1:
     debug "^2.6.8"
     resolve "^1.3.3"
 
+eslint-config-cozy-app@3.0.1:
```

Comment:
What is the change thant brings `eslint-config-cozy-app` in? 🤔

Reply:
https://github.com/cozy/cozy-libs/pull/1417#discussion_r785691827